hathor/p2p/states/ready.py

```diff
@@ -68,7 +71,15 @@ def __init__(self, protocol: 'HathorProtocol') -> None:
         })
 
         # Initialize sync manager and add its commands to the list of available commands.
-        self.sync_manager = NodeSyncTimestamp(self.protocol, reactor=self.reactor)
+        self.sync_manager: SyncManager
+        if protocol.protocol_version is ProtocolVersion.V1:
+            self.log.debug('loading sync-v1')
+            self.sync_manager = NodeSyncTimestamp(self.protocol, reactor=self.reactor)
```

Comment:
There is not difference between `protocol` and `self.protocol`, right? I saw super being called in the first line of the constructor (`super().__init__(protocol)`). But should we use only `self.protocol` here just for consistency?

Reply:
I think we should use `self.protocol` all along the code (except for the `super()`). In this case, we need to update the `self.log` as well.

Reply:
I agree. I'll make that change.

Reply:
Done.